app/renderer/js/notification/helpers.js

```diff
@@ -92,6 +134,10 @@ function setupReply(id) {
 	narrow.by_subject(id, { trigger: 'notification' });
 }
 
+window.addEventListener('load', () => {
```

Comment:
You should't put this under `load` because the while purpose of the function is to load bots before hand not after load event where notification could be created.

Reply:
The problem here is that, before ```load```, ```$``` is not defined.

Reply:
Oh, yeah i did not think about it you can ignore this then.

Reply:
@abhigyank, @cPhost  works great. I think the `load` listener should be added in `notification/index.js`.  Can you do that?